src/test/java/com/arpnetworking/metrics/impl/ApacheHttpSinkTest.java

```diff
@@ -36,62 +37,63 @@
 import java.util.List;
 import java.util.Map;
 import java.util.UUID;
+import java.util.concurrent.atomic.AtomicBoolean;
 
 /**
  * Tests for {@link ApacheHttpSink}.
  *
  * @author Brandon Arp (brandon dot arp at inscopemetrics dot com)
  */
-public class ApacheHttpSinkTest {
-    @Before
-    public void setUp() {
-        _wireMockServer = new WireMockServer(0);
-        _wireMockServer.start();
-        _wireMock = new WireMock(_wireMockServer.port());
-    }
-
-    @After
-    public void tearDown() {
-        _wireMockServer.stop();
-    }
+public final class ApacheHttpSinkTest {
 
     @Test
-    public void builderWithDefaults() {
+    public void testBuilderWithDefaults() {
         final Sink sink = new ApacheHttpSink.Builder().build();
         Assert.assertNotNull(sink);
         Assert.assertEquals(ApacheHttpSink.class, sink.getClass());
     }
 
     @Test
-    public void builderWithNulls() {
+    public void testBuilderWithNulls() {
         final ApacheHttpSink.Builder builder = new ApacheHttpSink.Builder();
         builder.setBufferSize(null);
         builder.setParallelism(null);
         builder.setUri(null);
         builder.setMaxBatchSize(null);
-        builder.setEmptyQueueSleepMillis(null);
+        builder.setEmptyQueueInterval(null);
         final Sink sink = builder.build();
         Assert.assertNotNull(sink);
         Assert.assertEquals(ApacheHttpSink.class, sink.getClass());
     }
 
     @Test
-    public void builderWithInvalidUri() {
+    public void testBuilderWithHttpsUri() {
         final ApacheHttpSink.Builder builder = new ApacheHttpSink.Builder();
-        builder.setUri("not a uri");
+        builder.setUri(URI.create("https://secure.example.com"));
+        final Sink sink = builder.build();
+        Assert.assertNotNull(sink);
+        Assert.assertEquals(ApacheHttpSink.class, sink.getClass());
+    }
+
+    @Test
+    public void testBuilderWithNotHttpUri() {
+        final ApacheHttpSink.Builder builder = new ApacheHttpSink.Builder();
+        builder.setUri(URI.create("ftp://ftp.example.com"));
         final Sink sink = builder.build();
         Assert.assertNotNull(sink);
         Assert.assertNotEquals(ApacheHttpSink.class, sink.getClass());
     }
 
     @Test
-    public void postSuccess() throws InterruptedException {
-        _wireMock.register(WireMock.post(WireMock.urlEqualTo(PATH))
+    public void testPostSuccess() throws InterruptedException {
+        WireMock.stubFor(WireMock.post(WireMock.urlEqualTo(PATH))
```

Comment:
I think we used the instance because using a static would bite us in concurrent tests.